reference/6/Microsoft.PowerShell.Utility/Sort-Object.md

```diff
@@ -313,11 +319,69 @@ the strings to integers. In the sample code, `[int]` converts the string to an i
 represents each string as it comes down the pipeline. The integer objects are sent down the pipeline
 to the `Sort-Object` cmdlet. `Sort-Object` sorts the integer objects in numeric order.
 
+### Example 9: Using stable sorts
+
+When you use the **Top**, **Bottom**, or **Stable** parameters, the sorted objects are delivered in
+the order they were received by `Sort-Object` when the sort criteria are equal. In this example, we
+are sorting the numbers one through 20 by the their value 'modulo 3'. The modulo value ranges from
+zero to two.
+
+```powershell
+PS> 1..20 |Sort-Object {$_ % 3}
+18
+3
+15
+6
+12
+9
+1
+16
+13
+10
+7
+4
+19
+11
+8
+14
+5
+17
+2
+20
+
++PS> 1..20 |Sort-Object {$_ % 3} -Stable
```

Comment:
There's a `+` in front of the row which will break the prompt.

Reply:
Fixed. It was a copy/paste from my prompt.